cli/src/api-key-success.html

```diff
@@ -1,254 +1,173 @@
+          fill="url(#paint0_linear_48_446)" />
+        <path
+          d="M185.414 343.189c-26.437-26.677-41.314-62.885-41.358-100.657-.035-31.07 9.968-61.105 28.198-85.668 6.55-8.825 19.187-8.769 26.911-.976l85.686 86.465.002-.002c26.438 26.677 41.315 62.884 41.358 100.656.041 34.962-12.63 68.614-35.432 94.632-3.045 3.475-8.325 3.466-11.575.187l-93.787-94.639-.003.002z"
+          fill="url(#paint1_linear_48_446)" />
+        <path
+          d="M476.062 512.532c-6.89 8.558-19.493 8.483-27.2.67l-52.926-53.657c-3.247-3.293-3.241-8.629.014-11.913l87.904-88.703c3.25-3.279 8.55-3.298 11.416.329 15.257 19.305 21.256 45.985 16.72 75.358-4.151 26.887-16.848 54.218-35.928 77.916z"
+          fill="url(#paint2_linear_48_446)" />
+        <path
+          d="M216.144 513.778c-7.706 7.813-20.309 7.888-27.199-.67-19.08-23.697-31.778-51.029-35.929-77.915-4.535-29.374 1.464-56.054 16.72-75.359 2.867-3.627 8.167-3.608 11.417-.329l87.904 88.703c3.255 3.285 3.261 8.621.013 11.913l-52.926 53.657z"
+          fill="url(#paint3_linear_48_446)" />
         <defs>
-          <clipPath id="clip0_4_2085">
-            <rect width="334" height="234" fill="white" />
-          </clipPath>
+          <linearGradient id="paint0_linear_48_446" x1="332.503" y1="149.563" x2="332.503" y2="443.522"
+            gradientUnits="userSpaceOnUse">
+            <stop stop-color="#9F87FF" />
+            <stop offset="1" stop-color="#8566FF" />
+          </linearGradient>
+          <linearGradient id="paint1_linear_48_446" x1="332.503" y1="149.563" x2="332.503" y2="443.522"
+            gradientUnits="userSpaceOnUse">
+            <stop stop-color="#9F87FF" />
+            <stop offset="1" stop-color="#8566FF" />
+          </linearGradient>
+          <linearGradient id="paint2_linear_48_446" x1="332.503" y1="353.019" x2="332.503" y2="519.292"
+            gradientUnits="userSpaceOnUse">
+            <stop stop-color="#DE99F6" />
+            <stop offset="1" stop-color="#D669FC" />
+          </linearGradient>
+          <linearGradient id="paint3_linear_48_446" x1="332.503" y1="353.019" x2="332.503" y2="519.292"
+            gradientUnits="userSpaceOnUse">
+            <stop stop-color="#DE99F6" />
+            <stop offset="1" stop-color="#D669FC" />
+          </linearGradient>
         </defs>
       </svg>
     </div>
+    <h1>Installation of the CLI is now complete</h1>
+
+    <p>
+      The Xata CLI is now linked to your account. An API key was added to
+      <code>~/.config/xata/credentials</code>.
+    </p>
+    <h2>Common commands</h2>
+<pre><code><span># List xata commands and get help</span>
+xata --help
+
+<span># Initialize Xata to run inside a project directory</span>
+xata init
 
-    <h1>Congratulations, you are all set!</h1>
+<span># Pull changes from the remote branch and regenerate code</span>
+xata pull</code></pre>
+    <a href="https://app.xata.io" class="btn">Back to Xata</a>
```

Comment:
Is this the URL they're most likely to want to go to?

 I'm wondering if users may benefit from being sent somewhere else to continue their "getting started" journey like:
 * https://xata.io/docs/getting-started/cli#commands
 * https://xata.io/docs/getting-started/installation#setting-up-necessary-xata-files-in-your-project
 * https://xata.io/docs/getting-started/intro
 * https://xata.io/docs/sdk/get#the-typescript-sdk-functions-for-querying

Reply:
I was thinking in a similar direction when I started. Here's some reasoning of why I kept it simple.

This page is shown after they've run `xata auth login` which they likely saw from instructions inside the app. The previous page they were on (https://github.com/xataio/frontend-next/pull/2697)  already links out to the installation pages within the docs (which I think is the best of the links you provided for someone this early).

This page follows and loads on a weird dead end URL from the client. Since they 100% had to go from the app to get to this weird URL, I think it's best to send them directly back to the app. The most common likely scenario (which I saw in sessions, and why I noticed this problem) is that they ran this procedure directly after seeing it in the app already (either from the snippet button, or similar on-boarding content in app). People got stuck in a loop of not finding their way back to their database.

Another sideways reason is that this page is unlikely to be checked by us often :cry: . As it exists now, it predates me and is over a year old! I'm worried about deep-linking from a page like that knowing that our docs tend to shift a bit. I'd prefer to keep it simple and funnel them to places we check a little more often (the app's flows).

If you want to introduce some mechanisms to track this better I'd welcome them help (in a separate PR). I admit I'm solving for a problem I'm seeing through a few dozen session views, rather than more methodical research. For now I consider this a small, incremental fix.

Reply:
@snide Thanks for this breakdown. I didn't realise this PR so closely lead on from the https://github.com/xataio/frontend-next/pull/2697 change, and that it's closing the loop on this specific existing user experience flow. It's really helpful to learn this extra context. 🙏 

I entirely agree with your assessment here re: deep linking, and that tracking this would be *awesome* but definitely better suited to a separate PR.